### PR TITLE
Add Rheinbahn Tram and Rename source

### PR DESCRIPTION
**Pull request by ChillLP**

### CODE REVIEW

**Comment by ChillLP** [comment]
Gib merge pls 🥺👉👈

File: line-colors.csv [hunk 1]
```diff
@@ -276,17 +276,24 @@ vrn-sbahn-rn,S33,db-regio-ag-mitte,4-801539-33,#833aa1,#ffffff,,pill
...
+vrr-rheinbahn,U71,,7-vrr070-71,#59c6f2,#ffffff,,rectangle
+vrr-rheinbahn,U72,,7-vrr070-72,#25b8c5,#ffffff,,rectangle
+vrr-rheinbahn,U73,,7-vrr070-73,#4465ad,#ffffff,,rectangle
+vrr-rheinbahn,U74,,7-vrr070-74,#2c2276,#ffffff,,rectangle
+vrr-rheinbahn,U75,,7-vrr070-75,#008fc2,#ffffff,,rectangle
+vrr-rheinbahn,U76,,7-vrr070-76,#0063af,#ffffff,,rectangle
+vrr-rheinbahn,U77,,7-vrr070-77,#7197cf,#ffffff,,rectangle
+vrr-rheinbahn,U78,,7-vrr070-78,#009adf,#ffffff,,rectangle
+vrr-rheinbahn,U79,,7-vrr070-79,#009a93,#ffffff,,rectangle
+vrr-rheinbahn,U83," ",7-vrr070-83,#1d3a8f,#ffffff,,rectangle
```
Comment (jheubuch) [inline line-colors.csv:289]:
```suggestion
vrr-rheinbahn,U83," ",7-vrr070-83,#1d3a8f,#ffffff,,rectangle
```

what is that " "? Why is that? :o

File: sources.json
Comment (jheubuch) [inline sources.json:1]: There are weird whitespaces in this file.
Do you use VS code? Could you install the "EditorConfig for VS Code" extension and then format the file again? :)

Thx!

**Review by jheubuch — CHANGES_REQUESTED** [review]

File: line-colors.csv (hunk 1 above)
Comment (ChillLP) [reply line-colors.csv:289]: interestingly enough it doesnt show that on my end, but ill try

**Comment by ChillLP** [comment]
🥺

**Review by jheubuch — APPROVED** [review]

**Pull request merged**